Source/BSN.IpTables.Cli/custom/Connect-BsnIPTablesCli.ps1

```diff
@@ -0,0 +1,22 @@
+function Connect-BsnIPTablesCli {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [BSN.IpTables.V1.Category('Uri')]
+        [System.String]
+        # Target Server Address
+        ${ServerAddress}
+    )
+
+    begin {
+        # Save the ServerAddress in a session variable
+        $env:ServerAddress = $ServerAddress
```

Comment:
I think we must to use scope environment variable to reduce chance of conflict

Reply:
I tryed about this and found finally some thing like this : 

[Connect.txt](https://github.com/BSVN/IpTables.Api/files/13806172/Connect.txt)
[Module.txt](https://github.com/BSVN/IpTables.Api/files/13806173/Module.txt)

but PSOutput always return empty I don't know why . 
and I think the best way to use virables is enviroment

my top searchs were : 

1. https://stackoverflow.com/questions/19678160/set-via-c-sharp-powershell-variable
2. 
3. https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_scopes?view=powershell-7.2#parent-and-child-scopes
4. 
5. https://stackoverflow.com/questions/56667578/how-can-i-get-and-set-variables-in-the-current-runspace-cmdlet-based-modules
6.

Reply:
@siadatism

Reply:
According to [this](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.4) environment variable could have three scopes: Machine, User and Process. The Process scope contains the environment variables available in the current process, or PowerShell session, which fits better for us.

So I think our current solution is good.